### DISCUSSION TITLE
Open questions to test on

### DISCUSSION THREAD

**Discussion by RussellDash332** [Schweinepriester/github-profile-achievements#10] [Q&A]
Open questions:
- Are you able to mark yourself as the answer, if OP is somebody else? Say, in a repo you have the rights to do that? @Schweinepriester can do this in other topics
- Do the discussions have to be in the Q&A category? This thread is not a Q&A so feel free to test yourselves here.
  - Edit: converted to Q&A

**Comment by ale-lit** [comment]
If the category is not Q&A it seems impossible to select an answer.

**Reply by RussellDash332** [reply]
> If the category is not Q&A it seems impossible to select an answer.

Answers the second one. Converting to Q&A

**Comment by tailsjs** [comment]
:)

**Comment by Schweinepriester** [comment]
For

> Are you able to mark yourself as the answer, if OP is somebody else? Say, in a repo you have the rights to do that?

it would be best if we had someone with a confirmed first answer, so that our minimal testing would instantly yield the achievement (because 2 accepted answers), seeing that "discussions answered" under the Highlights section doesn't necessarily correspond to the count for Galaxy Brain ([as was the case for me](https://github.com/Schweinepriester/github-profile-achievements/discussions/9#discussioncomment-2927413)).

So that person should have a test repo, where one of us asks a question, person responds and marks their own comment as the answer
=> Galaxy Brain or not?

Though per https://github.com/Schweinepriester/github-profile-achievements/issues/15

> 2.1 - Only answers in other people's questions selected by other participants are taken into account

the indication seems to be that it doesn't work.

**Reply by ale-lit** [reply]
I had a total of 8 counted answers in a mix (my answers to my questions, and other people's choices):
5 - my answers to my questions
3 - my answers to other people's questions

Everything is mixed. And they didn't give a reward.

-----------

Then I deleted all my answers to my questions, as well as some test answers in other people's test questions (for example, here), and in the end I only had 1 credited answer (under someone else's question).

-----------

Then I got the choice of my answer under someone else's question (https://github.com/ale-lit/visualizer/discussions/2#discussioncomment-2934043) and I was finally given a bounty )

That is, it only happened when my answer was №2. When he was No. 8, the award was not given.

**Comment by smir45** [comment]
Can I be able to get galaxy brain achievement?

**Comment by trentmillar** [comment]
🚜 🚚 🚗 🚓 🚌 🚒
